### PR TITLE
feat: added validation of min number in all the number fields

### PR DESCRIPTION
**Pull request by mayur-satav**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:

- Before: https://main--pricefx-eds--pricefx.hlx.live/
- After: https://feature-pai-39-be-search--pricefx-eds--pricefx.hlx.live/

### CODE REVIEW

**Review by muneeskumar-rajendran — DISMISSED** [review]

File: component-models.json [hunk 1]
```diff
@@ -1491,7 +1491,10 @@
         "name": "blogNumberOfArticles",
         "label": "Number Of Articles Display",
         "valueType": "number",
-        "value": 0
+        "value": 13,
+        "validation": {
+          "numberMin": 0
```
Comment (amogh-daryapurkar) [inline component-models.json]: @mayur-satav Can we change the min number to 3 or more? The 0 will not display the results.

File: component-models.json [hunk 2]
```diff
@@ -1530,7 +1533,9 @@
         "name": "numberOfArticles",
         "label": "Number Of Articles Display",
         "valueType": "number",
-        "value": 0
+        "validation": {
+          "numberMin": 0
```
Comment (amogh-daryapurkar) [inline component-models.json]: @mayur-satav Can we change the min number to 3 or more? The 0 will not display the results.

Comment (mayur-satav) [reply component-models.json]: we had call and decided to keep min 1

File: component-models.json (hunk 1 above)
Comment (mayur-satav) [reply component-models.json]: updated to 3

**Review by Vaitheesaran-nagalingam — APPROVED** [review]

**Pull request merged**